packages/@lightningjs/ui-components/src/components/TextBox/TextBox.js

```diff
@@ -61,7 +61,13 @@ export default class TextBox extends Base {
   }
 
   static get properties() {
-    return ['content', 'fixed', 'marquee', ...InlineContent.properties];
+    return [
+      'content',
+      'fixed',
+      'marquee',
+      ...InlineContent.properties,
```

Comment:
we should list the destructured item first, unless there's a specific reason not to(a property we explicitly we want overridden or something)